src/modes/map/map_objects/map_escape_point.cpp

```diff
@@ -21,9 +21,12 @@ namespace vt_map
 namespace private_map
 {
 
-const std::string ESCAPE_POINT_ACTIVATION_SOUND = "data/sounds/escape_activate.ogg";
-const std::string ESCAPE_POINT_ACTIVE_PARTICLE_EFFECT = "data/visuals/particle_effects/active_escape_point.lua";
-const std::string ESCAPE_POINT_INACTIVE_PARTICLE_EFFECT = "data/visuals/particle_effects/inactive_escape_point.lua";
+const std::string ESCAPE_POINT_ACTIVATION_SOUND =
+    "data/sounds/escape_activate.ogg";
```

Comment:
For your info,
I'm sorry, but this kind of change don't fit IMHO. I really dislike breaking variable declaration that way, just because you hard wrap on 80 characters, or any line split because it is 84 characters and the like.
I'll remove that kind of changes in any of your PRs.